results-mieb/jinaai__jina-clip-v1/1cbe5e8b11ea3728df0b610d5453dfe739804aa9/MSCOCOI2TRetrieval.json

```diff
@@ -0,0 +1,158 @@
+        "nauc_precision_at_1000_std": 0.49837165139806966,
+        "nauc_precision_at_100_diff1": 0.16399269478318815,
+        "nauc_precision_at_100_max": 0.3174441756422564,
+        "nauc_precision_at_100_std": -0.2719045017872643,
+        "nauc_precision_at_10_diff1": 0.1834674594026287,
+        "nauc_precision_at_10_max": 0.2475111637901415,
+        "nauc_precision_at_10_std": -0.36964525963118233,
+        "nauc_precision_at_1_diff1": 0.3620563318972254,
+        "nauc_precision_at_1_max": 0.22304702323816303,
+        "nauc_precision_at_1_std": -0.39325847713016515,
+        "nauc_precision_at_20_diff1": 0.18190648500233034,
+        "nauc_precision_at_20_max": 0.2543371579459487,
+        "nauc_precision_at_20_std": -0.3684848720360581,
+        "nauc_precision_at_3_diff1": 0.2216308838348382,
+        "nauc_precision_at_3_max": 0.23889515868420422,
+        "nauc_precision_at_3_std": -0.3624204875957743,
+        "nauc_precision_at_5_diff1": 0.19295589426112914,
+        "nauc_precision_at_5_max": 0.24668879287833217,
+        "nauc_precision_at_5_std": -0.36421771939542763,
+        "nauc_recall_at_1000_diff1": -0.04813353912994217,
+        "nauc_recall_at_1000_max": 0.47849854227402444,
+        "nauc_recall_at_1000_std": 0.5228805808037352,
+        "nauc_recall_at_100_diff1": 0.16501981261932044,
+        "nauc_recall_at_100_max": 0.3183331361282548,
+        "nauc_recall_at_100_std": -0.2726147056630754,
+        "nauc_recall_at_10_diff1": 0.1835675928057196,
+        "nauc_recall_at_10_max": 0.24766746744342197,
+        "nauc_recall_at_10_std": -0.3696577837871595,
+        "nauc_recall_at_1_diff1": 0.3621079914040161,
+        "nauc_recall_at_1_max": 0.22314949459519212,
+        "nauc_recall_at_1_std": -0.39289670930715714,
+        "nauc_recall_at_20_diff1": 0.18207077982715833,
+        "nauc_recall_at_20_max": 0.254625446742885,
+        "nauc_recall_at_20_std": -0.36840440513586076,
+        "nauc_recall_at_3_diff1": 0.2218119797611259,
+        "nauc_recall_at_3_max": 0.23903410739410114,
+        "nauc_recall_at_3_std": -0.3625477824595922,
+        "nauc_recall_at_5_diff1": 0.1931427010978174,
+        "nauc_recall_at_5_max": 0.24689867912760904,
+        "nauc_recall_at_5_std": -0.364330180957868,
+        "ndcg_at_1": 0.544,
+        "ndcg_at_10": 0.48279,
+        "ndcg_at_100": 0.6064,
+        "ndcg_at_1000": 0.63736,
+        "ndcg_at_20": 0.53283,
+        "ndcg_at_3": 0.47129,
+        "ndcg_at_5": 0.41457,
+        "precision_at_1": 0.544,
+        "precision_at_10": 0.24992,
+        "precision_at_100": 0.04259,
+        "precision_at_1000": 0.00496,
+        "precision_at_20": 0.15413,
+        "precision_at_3": 0.45027,
+        "precision_at_5": 0.37588,
+        "recall_at_1": 0.10886,
+        "recall_at_10": 0.50001,
+        "recall_at_100": 0.85215,
+        "recall_at_1000": 0.99216,
+        "recall_at_20": 0.6168,
+        "recall_at_3": 0.27025,
+        "recall_at_5": 0.376
```

Comment:
Not matching the scores claimed by the paper:
- MSCOCO Text Retr. R@1: 0.5544
- MSCOCO Text Retr. R@5: 0.7904